tests/test_vqe.py

```diff
@@ -582,6 +716,152 @@ def test_passing_kwargs(self, coeffs, observables, expected):
             assert qnode.h == 123
             assert qnode.order == 2
 
+    def test_optimize_outside_tape_mode(self):
+        """Test that an error is raised if observable optimization is requested outside of tape
+        mode."""
+        if qml.tape_mode_active():
+            pytest.skip("This test works with tape mode disabled")
```

Comment:
```suggestion
            pytest.skip("This test is only intended for non-tape mode")
```
Makes it a bit easier to interpret?